_layouts/default.html

```diff
@@ -8,7 +8,8 @@
         {{ content }}
       </div>
     </div>
-    </div>	
+    </div>
```

Comment:
Why were these 2 extra `</div>` added?

Reply:
I added them to separate the footer from the `#wrapper`.

The wrapper was consuming the footer as well.

Reply:
The confusion is arising because there are 2 wrappers.
A `.wrapper` and a `#wrapper`.

Reply:
@akshatnitd Is it fine?